lib/internal/bootstrap/pre_execution.js

```diff
@@ -221,6 +222,30 @@ function initializeDeprecations() {
                                 'process.binding() is deprecated. ' +
                                 'Please use public APIs instead.', 'DEP0111');
   }
+
+  let _process = process;
```

Comment:
can you just add a comment explaining why we have these getters/setters

Reply:
Sure, added.